tap_ilevel/sync.py

```diff
@@ -538,6 +539,7 @@ def __process_periodic_data_calcs(req_state, scenario_name='Actual', currency_co
                                     else:
                                         value_numeric = None
                                     if value == 'No Data Available':
+                                        LOGGER.info(transformed_record.get("sd_parameters").get("end_of_period").get("value"))
```

Comment:
Can you improve this log message to make more sense to someone reading the logs?